package.json

```diff
@@ -2,13 +2,6 @@
   "name": "data-sync-server",
   "version": "1.0.0",
   "description": "GraphQL based server for syncing data between clients",
-  "main": "index.js",
-  "scripts": {
-    "test": "standard --fix && mocha",
-    "start": "node ./server.js",
-    "start-dev": "SCHEMA_FILE=./examples/schema.example.graphql DATA_SOURCES_FILE=./examples/data-sources.example.json RESOLVER_MAPPINGS_FILE=./examples/resolver-mappings.example.json QUERY_FILE=./examples/query.example.graphql nodemon ./server.js",
-    "format": "standard --fix"
-  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/aerogear/aerogear-data-sync-server.git"
```

Comment:
would you mind changed the repo url here

Reply:
I see the reason but I was playing around some of the code and one of things that happens every once in a while is that standard encounters an issue that cannot be automatically fixed. One example which occurs often enough in local dev is unused variables.

When this happens the whole thing crashes a bunch of times before failing. I think it's much handier if we simply include linting as part of precommit hooks and as part of CI checks. Happy to keep it though if you prefer

Reply:
sure!